public/app/core/utils/ticks.ts

```diff
@@ -25,3 +25,43 @@ export function tickStep(start: number, stop: number, count: number): number {
 
   return stop < start ? -step1 : step1;
 }
+
+export function getScaledDecimals(decimals, tick_size) {
```

Comment:
I see that this is code for generating ticks from Flot. It's a pity we had to duplicate it but looking at the flot code, this is embedded in the setupTickGeneration function and is not really reusable.